Makefile

```diff
@@ -51,7 +51,8 @@ ADDON_BINDATA_DIR = $(CURDIR)/$(BUILD_DIR)/bindata
 ADDON_ASSET_FILE = $(ADDON_BINDATA_DIR)/addon_assets.go
 
 # Setup for the docs tasks
-IMAGE_UID ?= 1000
+DOCS_BUILDER_IMAGE = minishift/minishift-docs:1.0.0
```

Comment:
Ideally it should be called `minishift-docs-builder` isn't it?

Reply:
Maybe. Both works for me.

Reply:
@LalatenduMohanty how strongly do you feel about this? I'd like to sort everything out asap.

Reply:
@hferentschik Not very strongly. I can live with current name also.